testing/ods/common.sh

```diff
@@ -8,8 +8,8 @@ ODS_CATALOG_VERSION="quay.io/modh/qe-catalog-source"
 ODS_CATALOG_IMAGE_VERSION="v1100-6"
 
 ODS_CI_TEST_NAMESPACE=loadtest
-ODS_CI_REPO="https://github.com/openshift-psap/ods-ci.git"
-ODS_CI_REF="multiuser"
+ODS_CI_REPO="https://github.com/fcami-test/psap-ods-ci.git"
+ODS_CI_REF="from_fcami"
```

Comment:
Could you switch to https://github.com/fcami/ods-ci/ please? the branch name is the same.

Reply:
done for the switch,
note that this is part of a `WIP` commit, it will be removed once the rebase is validated and https://github.com/openshift-psap/ods-ci/pull/2 is merged